packages/expo-cli/src/credentials/provider/index.ts

```diff
@@ -0,0 +1,3 @@
+export * from './provider';
```

Comment:
I really don't like `index.ts` files like that. 
1. What if there'll be someone who (by an accident) will add a function with the same name as one in another files?
2. You're saving an extra import line but you need to waste some time when you're looking for the function definition because you need to open a file and then notice that the actual definition is somewhere else.